qcodes/tests/dataset/test_converters.py

```diff
@@ -77,18 +82,19 @@ def test_construct_currect_rundesciber_from_v0(some_paramspecs):
 
     rds2 = from_dict_to_current(v0)
```

Comment:
i got confused with the number in the name of this variable, perhaps this is better :) same for rds1 above
```suggestion
    rds_from_func = from_dict_to_current(v0)
```